### DISCUSSION TITLE
Multiple Versions of the same book/series?

### DISCUSSION THREAD

**Discussion by susan3333333** [advplyr/audiobookshelf#1437] [Q&A] [answered]
I am sure I saw something about this here already, but I can't find it now.

What is the best way to handle a book , or a series, where there is more than one version?

**Comment by hobesman** [comment, marked as answer]
You're free to post here but the conversation on discord is much more active and more likely to get responses. Also more likely to find answers to questions that were already asked.

https://discord.gg/pJsjuNCKRq

But to get to your question, there are a few options.

First, any books in separate folders will be seen by abs as separate books, so that's the first step. Then once they're separate books, my preference is to include the version info in the book name and/or series name.

So for example I have two versions of Lord of the Rings by two different narrators. So I named the books "The Fellowship of the Ring (Andy Serkis)" and "The Fellowship of the Ring (Rob Inglis)". I also named the series accordingly, so I have one series called "Lord of the Rings (Andy Serkis)" and another series called "Lord of the Rings (Rob Inglis)". By keeping them as two separate series, if I finish listening to one book, it recommends the next one with the same narrator, rather than recommending the other version of the same book over again but with a different narrator.

**Reply by susan3333333** [reply]
Thank you, I like the series idea.
I have Fred, the Vampire Accountant normal and Graphic Audio versions, I think the series solution is perfect for that :)
